plugins/autoloader/prism-autoloader.js

```diff
@@ -10,8 +10,19 @@
 
 	var ignored_language = 'none';
 
+	var script = document.getElementsByTagName('script');
+	script = script[script.length - 1];
+	var languages_path;
+	if(script.hasAttribute('data-autoloader-path')) {
+		var path = script.getAttribute('data-autoloader-path').trim();
+		if(path != '') {
+			languages_path = path;
+		}
+	} else {
+		path = script.src.replace(/[\w\-]+\.js$/, 'components/');
```

Comment:
Shouldn't you set `languages_path` here instead?
Also, you don't need to escape `\-` in that case.